input_name.py

```diff
@@ -0,0 +1,101 @@
+from background import Background
+from constants import *
+from button import Button
+from pygame_textinput import TextInputVisualizer, TextInputManager
+
+
+class InputName(pygame.sprite.Sprite):
+    #Constructor
+    def __init__(self):
+        super().__init__()
+        #Instruction and re-enter name strings
+        self.instruction_string = "Please input your name(must be between 1 and 15 characters) below:"
+        self.reenter_string = "Your name must be at least 1 character, please try again"
```

Comment:
Always nice to have messages to tell the user what went wrong

Reply:
Yeah I thought it was a nice touch!